application/controllers/admin/questions.php

```diff
@@ -1736,10 +1736,14 @@ function($attribute) { return \viewHelper::flatten($attribute); },
                 ); 
                 foreach ($oLabelSet->labels as $oLabel) {
                     if($oLabel->language === $sLanguage) {
-                        $aResult[$sLanguage]['labels'][] = array_map(
-                            function($attribute) { return \viewHelper::flatten($attribute); },
-                            $oLabel->attributes
-                        ); 
+                        $aLabels = $oLabel->attributes;
+                        array_walk(
+                            $aLabels,
+                            function(&$attribute, $key) {
```

Comment:
My opinion : leave this to model and html5 system …
Donc't filter in controller :)